pontoon/teams/views.py

```diff
@@ -77,6 +78,27 @@ def ajax_info(request, locale):
     })
 
 
+@require_POST
+@permission_required_or_403('base.can_manage_locale', (Locale, 'code', 'locale'))
+@transaction.atomic
+def ajax_update_info(request, locale):
+    team_description = request.POST.get('team_info', None)
+
+    # Allow <p> and <br> tags
+    tags = bleach.ALLOWED_TAGS[:]
```

Comment:
I assume you want to make a copy of ``ALLOWED_TAGS`` here. It is generally more readable to do ``list(bleach.ALLOWED_TAGS)`` instead. The usage of ``[:]`` is quite cryptic and considered bad practice.